src/containers/MyNdla/Folders/FolderActions.tsx

```diff
@@ -222,7 +222,11 @@ const FolderActions = ({
           ? t('myNdla.folder.sharing.button.goTo')
           : t('myNdla.folder.sharing.button.preview'),
       onClick: () => {
-        navigate(sharedFolderLinkInternal(selectedFolder.id));
+        if (selectedFolder.status === 'shared') {
+          navigate(sharedFolderLinkInternal(selectedFolder.id));
+        } else {
+          navigate(previewLink(selectedFolder.id));
+        }
```

Comment:
Tviler på at det er her det er brukket, men jeg blir redirecta til test når jeg prøver dette.

Reply:
Er ganske sikker på at det ikke er det skjer, iallefall. Avhengig av status på mappa, får du enten `/folder/:folderId` (hvis mappa er delt) eller `/minndla/folders/preview/:folderId` (hvis mappa ikke er delt). Men det har opprinnelig vært lenka til test for forhåndsvisning-lenka (som jeg har antatt det er en grunn te) pga bruk av `config.ndlaFrontendDomain` i preview-lenke-funksjonen. Jeg rydda litt opp i det, til istedet å bruke relative lenkestier (men uten å erstatte den eksisterende, sida den også var brukt til f.eks. kopi av delt mappe-lenke).
Så nå at det var 1 forhåndsvisnings-knapp som fortsatt brukte den gamle, så har oppdatert den til istedet å bruke Internal-funksjonen med relativ sti også (håper og trur at det var den du brukte, men si ifra hvis du fortsatt har samme opplevelse).